azure/marketplace/mainTemplate.json

```diff
@@ -293,7 +293,7 @@
     "vTrue": true,
     "bootstrapPublicIpDnsLabel": "[concat('bootstrapdns', uniqueString(resourceGroup().id))]",
     "sshKeyPath": "[concat('/home/', parameters('bootstrapAdminUsername'), '/.ssh/authorized_keys')]",
-    "bootstrapScriptUrl": "https://raw.githubusercontent.com/IBM/zmodstack-deploy/dev/azure/scripts/bootstrap.sh",
+    "bootstrapScriptUrl": "https://raw.githubusercontent.com/IBM/zmodstack-deploy/main/azure/scripts/bootstrap.sh",
```

Comment:
This will encounter the same issues we have with non-parameterized deployments in AWS... as we work on the `dev` branch and attempt to add/remove any logic, our changes from the `dev` branch won't be used.

Reply:
What I thought is the developer has to manually edit the URL with their `feature` branch name for testing any changes in their `feature` branch.
Are there any better approaches, such as dynamically changing the URL for the ARM Variable `bootstrapScriptUrl`?

Reply:
Regarding making `bootstrapScriptUrl` as an Ansible Variable, actually the ARM Template is executed at first with its default `bootstrapScriptUrl` from `main` branch, which in turn executes the Ansible playbooks and does the jinja templating alongside. Thus it doesn't seems feasible to implement the branch swap in `bootstrapScriptUrl`  much before the execution of Ansible playbooks.